### ISSUE TITLE
Particals are invisible 

### ISSUE THREAD

**Issue by produxe** [Vazkii/Botania#4297]
### Mod Loader

Fabric

### Minecraft Version

1.18.1

### Botania version

1.18.1-429

### Modloader version

Fabric 0.13.3  Fabric Api 0.46.6

### Modpack info

https://www.curseforge.com/minecraft/modpacks/infinity-fabric

### The latest.log file

dont know how

### Issue description

Cant see any particals from mana and cant see the center of the pylons. Ive checked the config and it seems to be enabled

### Steps to reproduce

Download infinity mod pack to server and see

**Comment by Pultex** [comment]
shaders of any sort do this I had to turn them off

**Comment by produxe** [comment]
Yes that helped partly, i can see sparkes and pylons now but i still cant see the visual transfer from then mana spreaders

**Comment by blagdorfinguy** [comment]
Bump.

Present in 1.20.1, Botania version 1.20.1-441

Disabling shaders in config helps with pylons (see #3951) but does not change the rendering of particles with alpha channels.
Running Oculus 1.6.13 and Embeddium 0.2.13
